src/Aspire.Hosting.Azure.ServiceBus/ServiceBusEmulatorContainerImageTags.cs

```diff
@@ -20,6 +20,6 @@ internal static class ServiceBusEmulatorContainerImageTags
     /// <remarks>mssql/server</remarks>
     public const string SqlServerImage = "mssql/server";
 
-    /// <remarks>2022-latest</remarks>
-    public const string SqlServerTag = "2022-latest";
+    /// <remarks>2025-latest</remarks>
+    public const string SqlServerTag = "2025-latest";
```

Comment:
```suggestion
    /// <remarks>2022-latest</remarks>
    public const string SqlServerTag = "2022-latest";
```
per suggestion from @davidfowl. Do we have an issue for this?